rosserial_tivac/src/ros_lib/startup_gcc.c

```diff
@@ -0,0 +1,490 @@
+/*
+ * Copyright (c) 2015, Robosavvy Ltd.
```

Comment:
No I'm not clear about that. It's a case of copy pasties.
It's based on the TI-supplied Energia file https://github.com/energia/Energia/blob/master/hardware/lm4f/cores/lm4f/startup_gcc.c 
although several changes were made.

I'll retain the original header on the next commit.